### DISCUSSION TITLE
Give the NAT more options to change ranked beatmaps

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#30000] [Ideas] [answered]
I have submitted two reports to the NAT for an incorrect flashing lights warning 
(https://bn.mappersguild.com/message?report=668d6b0166bc40d3c76fe9f8) 
and for an incorrect map title 
(https://bn.mappersguild.com/message?report=66e429a679b0dc99328f9234).
Both reports have been found as correct, however the NAT doesn't have the ability to change such mistakes.
Therefore I would like to propose giving the NAT the ability to change more things concerning ranked beatmaps without those maps having to lose their ranked status so that we can fix those mistakes without the fixing itself becoming a nuisance for the mapper.

**Comment by bdach** [comment]
As things stand, changing metadata after the beatmap has been ranked requires it to be *un*ranked briefly to fix the metadata mistakes, which wipes scores. This is why minor issues in metadata (which these *are*) will likely not be actioned upon.

In the past a workaround that doesn't require unranks *was* used, but it is no longer allowed as it was deemed to be a crutch, and that the ranking process should be kept strict.

**Reply by ghost** [reply]
Could you elaborate what you mean by "the ranking process should be kept strict.". 
After all this is a case where the ranking process wasn't upholded correctly and should in my eyes be fixed to assure a certain level of quality of the ranking section.

**Reply by bdach** [reply]
The workaround that was used in the past was not a 100% safe operation either, it ran the risk of being a destructive one. However it kept getting used because of the perceived high likelihood that it would be "fine this time".

Requiring these sorts of changes to escalate to full unranks gives them the proper gravity.

**Reply by peppy** [reply, marked as answer]
In addition, making changes like this post-ranking invalidates replays amongst other things. And we really don't have these kinds of flows handled correctly in the client.

It's best to just live with slightly incorrect metadata as an "oops, well, it happened" for now.
